doc/classes/TextEdit.xml

```diff
@@ -1362,6 +1362,7 @@
 		</member>
 		<member name="syntax_highlighter" type="SyntaxHighlighter" setter="set_syntax_highlighter" getter="get_syntax_highlighter">
 			Sets the [SyntaxHighlighter] to use.
```

Comment:
While you're here:
This is worded like a method, a setter, for some reason. And repeating the property's type makes it a bit annoying to localise.
```suggestion
			The syntax highlighter to use.
```